src/loaders/gal4_load.c

```diff
@@ -68,9 +68,9 @@ static int get_main(struct module_data *m, int size, HIO_HANDLE *f, void *parm)
 	struct xmp_module *mod = &m->mod;
 	char buf[64];
 	int flags;
-	
+
 	hio_read(buf, 1, 64, f);
-	strncpy(mod->name, buf, 63);	/* ensure string terminator */
+	snprintf(mod->name, XMP_NAME_SIZE, "%.*s", 63, buf);
```

Comment:
63 should be 64 here.

Reply:
OK confused.

Reply:
I added the 63 to these mostly to avoid another annoying warning GCC emits re: snprintf truncation. It doesn't show up when I remove that so I'll just remove both of those.

Reply:
OK: https://stackoverflow.com/questions/7899119
Never used precision myself..

Reply:
I removed .* and 63 from both of the Galaxy loader snprintfs.